Makefile

```diff
@@ -57,7 +57,7 @@ help:
 .gopathok:
 ifeq ("$(wildcard $(GOPKGDIR))","")
 	mkdir -p "$(GOPKGBASEDIR)"
```

Comment:
Shouldn't this line be changed too?

Reply:
Unless I'm reading it wrong, no. We want `$GOPATH/src/github.com/kubernetes-incubator` to be created, then create a link in that directory